### PR TITLE
Updated conformance test scripts and workflow

### PR DESCRIPTION
**Pull request by Yasasr1**
Changes to workflow

- Added two separate parameters for sending google chat message and sending email
- Added a separate step to display test results

Changes to scripts and configurations

- updated configure_is.py to return error if a HTTP error occurs
- Removed send_notification.py and added two separate scripts to handle sending email and google chat messages  
- Modified DOB format when using SCIM API to add user birthday

### CODE REVIEW

File: oidc-conformance-tests/README.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ OIDC conformance test workflow can be used to for this purpose.
 2. Click on OIDC conformance test workflow
 3. Click on Run workflow
 4. You need to provide the tag name of the product-is version you want to test (default value is "v5.12.0-m6")
-5. Set send google chat notification to false if you don't want a summary of test results to be sent to a configured google chat group. (default value is true)
-6. Click on Run workflow
-7. After tests are completed you can view test results on the test summary page
-8. Two types of artifacts are saved after the test is completed
+5. Set 'Send test results to email' to 'no' if you don't want to send test summary to a list of pre-configured email addresses. (default is 'yes')
```
Comment (mefarazath) [inline oidc-conformance-tests/README.md:10]: Can we set the default to no for this option too

File: oidc-conformance-tests/basic/config/service_provider1_config.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
         },
         "advancedConfigurations": {
             "discoverableByEndUsers": false,
-            "skipLoginConsent": false
+            "skipLoginConsent": true
```
Comment (mefarazath) [inline oidc-conformance-tests/basic/config/service_provider1_config.json:37]: skipping consent as a temporary solution is fine. But we need to investigate and fix this.

**Review by mefarazath — APPROVED** [review]

File: oidc-conformance-tests/README.md (hunk 1 above)
Comment (Yasasr1) [reply oidc-conformance-tests/README.md]: updated

File: oidc-conformance-tests/README.md [hunk 3]
```diff
@@ -7,10 +7,11 @@ OIDC conformance test workflow can be used to for this purpose.
...
 4. You need to provide the tag name of the product-is version you want to test (default value is "v5.12.0-m6")
-5. Set send google chat notification to false if you don't want a summary of test results to be sent to a configured google chat group. (default value is true)
-6. Click on Run workflow
-7. After tests are completed you can view test results on the test summary page
-8. Two types of artifacts are saved after the test is completed
+5. Set 'Send test results to email' to 'yes' if you want to send test summary to a list of pre-configured email addresses. (default is 'no')
+6. Set 'Send test results to google chat' to 'yes' if you also want to send test summary to a pre-configured google chat group. (default is 'no')
+7. Click on Run workflow
+8. After tests are completed you can view test results on the test summary page
+9. Two types of artifacts are saved after the test is completed
```
Comment (dewniMW) [inline oidc-conformance-tests/README.md:14]:
```suggestion
9. Two types of artifacts are saved after the test execution is completed
```

**Review by dewniMW — APPROVED** [review]

File: oidc-conformance-tests/basic/config/service_provider1_config.json (hunk 2 above)
Comment (dewniMW) [reply oidc-conformance-tests/basic/config/service_provider1_config.json:37]: What is the issue with skipping login consent?

Comment (Yasasr1) [reply oidc-conformance-tests/basic/config/service_provider1_config.json:37]: It's a issue with the conformance suite. Maybe a bug in the HtmlUnit browser they use. It didn't register redirects from the IS

Comment (mefarazath) [reply oidc-conformance-tests/basic/config/service_provider1_config.json:37]: Let's create an issue on both our side and confirmance suite repo to track this

**Pull request merged**